### PR TITLE
docs: added example for awscc_codegurureviewer_repository_association

### PR DESCRIPTION
**Pull request by quixoticmonk**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

File: templates/resources/codegurureviewer_repository_association.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### S3 as the repository to associate CodeGuru Reviewer with.  The bucket name must start with the prefix codeguru-reviewer-*
+
+{{ tffile (printf "examples/resources/%s/codegurureviewer_repository_association_s3.tf" .Name)}}
+
+tags["codeguru-reviewer"]
```
Comment (wellsiau-aws) [inline templates/resources/codegurureviewer_repository_association.md.tmpl:18]:
```suggestion
```

I dont believe this particular line is needed, @quixoticmonk can you confirm?

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
Please see my comment on the review, thanks!

File: templates/resources/codegurureviewer_repository_association.md.tmpl (hunk 1 above)
Comment (quixoticmonk) [reply templates/resources/codegurureviewer_repository_association.md.tmpl]: You are right. Thanks for catching that. The note in the markdown should handle that tag reference. Was copied over by mistake.

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thank you!

**Comment by marcosentino** [comment]
@quixoticmonk Thank you for the contribution!

**Pull request merged**